### PR TITLE
Kafka: Docs for kafka-mesos role in installation procedure

### PR DESCRIPTION
**Pull request by steel4ge**

### CODE REVIEW

File: docs/features/create_mesos_cluster.rst [hunk 1]
```diff
@@ -153,7 +153,19 @@ Step-by-step Guide
 
         Pi is roughly 3.14336
 
-8. Verify Kafka-mesos utility:
+8. Before kakfa-mesos application is installed, a number of created and running bockers can be changed to any quantity.
```
Comment (OleksandrBerchenko) [inline docs/features/create_mesos_cluster.rst]: brockers

Comment (OleksandrBerchenko) [inline docs/features/create_mesos_cluster.rst]: can be changed.

File: docs/features/create_mesos_cluster.rst [hunk 2]
```diff
@@ -153,7 +153,19 @@ Step-by-step Guide
...
+
+        Edit variable kafka_brokers: "<number>" in roles/kafka-mesos/defaults/main.yml
+
+   In order to enable brocker installation process variable enable_client_install should be set to "true".
+   In case of "false" value, no any brockers will be installed.
```
Comment (OleksandrBerchenko) [inline docs/features/create_mesos_cluster.rst]: no brockers

**Comment by ohal** [comment]
@ test this please

**Comment by ohal** [comment]
@ ok to test

**Comment by ohal** [comment]
failed with:

```

TASK: [wait for host boot] **************************************************** 
fatal: [host-02 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-06 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-04 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-01 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-03 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-05 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-07 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined
fatal: [host-08 -> 127.0.0.1] => One or more undefined variables: 'ansible_ssh_host' is undefined

```

**Pull request merged**